CodeGeneration/Sources/SyntaxSupport/Node.swift

```diff
@@ -137,3 +141,23 @@ public class Node {
     assert(!isSyntaxCollection || elementName != nil || element != "")
   }
 }
+
+extension Node: Equatable {
```

Comment:
Needed to add `Equatable` so we can do this: https://github.com/apple/swift-syntax/pull/1136/files#r1050450686


If you have any other suggestion or idea @ahoppen please enlighten me 😁

Reply:
In case we really need this (which I don’t think), we should use the automatically synthesized `Equatable` conformance. This is too fragile to maintain.